network_insight_sdk_generic_datasources/common/constants.py

```diff
@@ -33,6 +33,7 @@
 RESULT_WRITER_KEY = 'result_writer'
 PATH_KEY = 'path'
 GENERATION_DIRECTORY_KEY = 'generation_directory'
+FILE_INPUT_DIRECTORY_KEY = 'file_input_directory'
```

Comment:
OFFLINE_OUTPUT_COLLECTION_DIRECTORY_KEY='offline_output_collection_directory'
Anything you want to use ?

Reply:
OFFLINE_COMMAND_RESULTS_DIRECTORY_KEY